.github/workflows/whitespace.yml

```diff
@@ -0,0 +1,47 @@
+name: Whitespace
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
+    types:
+      - created
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    env:
+     fix-whitespace-ver: '0.0.8'
+
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout sources
+
+    - name: Create directory for binary
+      run: |
+        mkdir -p $HOME/.local/bin
+
+    - uses: actions/cache@v2
```

Comment:
The latest version of the `checkout` and `cache` actions is `v3` so you can update it already 🙂

Reply:
Done!